core/src/types.ts

```diff
@@ -266,3 +266,53 @@ export declare interface DebtAuctionEnriched extends DebtAuctionActive {
 }
 
 export declare interface DebtAuctionTransaction extends DebtAuctionEnriched, CompensationAuctionTransactionFees {}
+
+type CollateralType = CollateralConfig['title'];
+export declare interface VaultCollateralType {
+    collateralType: CollateralType;
+}
+
+export declare interface LiquidationLimit {
+    maximumAmountAuctionedCollateral: BigNumber;
+    currentlyAuctionedCollateral: BigNumber;
+}
```

Comment:
- [ ] Now you're missing the type for global (non-collateral) limits
- [ ] Naming can be simpler and include currency
```suggestion
export declare interface LiquidationLimits {
    maximumProtocolDebtDai: BigNumber;
    maximumCollateralDebtDai: BigNumber;
    currentProtocolDebtDai: BigNumber;
    currentCollateralDebtDai: BigNumber;
}
```